background.js

```diff
@@ -12,6 +12,8 @@ chrome.webRequest.onHeadersReceived.addListener(({ responseHeaders, url }) => {
     for (let p of cspAllowAll) {
       csp.value = csp.value.replace(`${p}`, `${p} * blob: data:`); // * does not include data: URIs
     }
+    // Discord doesn't even specify a manifest CSP so we create our own
+    csp.value += 'manifest-src * blob: data:'
```

Comment:
Please keep consistent syntax as with the rest of the file (;)

Reply:
as this was the last entry in the file, the terminator didn't matter that much, but fixed, made it inline with all other CSP tags

Reply:
Syntax of the JS, not in the CSP

Reply:
fixed